cloud/ec2_util.go

```diff
@@ -216,19 +224,19 @@ func timeTilNextEC2Payment(h *host.Host) time.Duration {
 	return time.Second
 }
 
-// UsesHourlyBilling checks if a distro name to see if it is billed hourly,
-// and returns true if so (for example, most linux distros are by-the-minute).
```

Comment:
Good catch on the per-minute vs per-second

Reply:
thank you! 
I wonder if it used to be per-minute and that also changed or if we always had it wrong.